GitCommands/Remote/GitRemoteManager.cs

```diff
@@ -70,18 +70,21 @@ public void ConfigureRemotes(string remoteName)
         {
             var module = GetModule();
             var localConfig = module.LocalConfigFile;
+            var moduleRefs = module.GetRefs(tags: false, branches: true);
 
-            foreach (var remoteHead in module.GetRefs(true, true))
+            foreach (var remoteHead in moduleRefs)
             {
-                foreach (var localHead in module.GetRefs(true, true))
+                if (!remoteHead.IsRemote ||
+                    !StringExtensions.Contains(remoteHead.Name, remoteName))
```

Comment:
You can call your extension method as if it were an instance method of the `this` parameter. 

In this case you could write:

```
remoteHead.Name.Contains(remoteName, StringComparison.InvariantCultureIgnoreCase)
```

which reads clearly.

Reply:
Hello, I did it that way to make it clear to the reader of the function that i'm not calling the extension provided by the framework but the extension that i wrote myself.

Reply:
In this case I think the extension method is quite obvious and that most people wouldn't even notice it wasn't a method on string itself. We could likely use this in many other places in the code.